install/openshift-apiserver/static-config.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@sttts are we going to render these?

Reply:
> @sttts are we going to render these?

No.  openshift-apiserver never runs statically.

Reply:
Where is the file used?

Reply:
> Where is the file used?

it's not, cleaning up cruft now.